### PR TITLE
Initial version of generators

### PR DESCRIPTION
**Pull request by alexjp**
Hi,

I created some generators using json.gen.resource namespace.

Currently I tested them with #21 example and they worked for index, create, update and show.

If :+1:  for you, I will do the tests next ( I tested them manually ).

Thanks !

### CODE REVIEW

**Comment by jeregrine** [comment]
Hey @alexjp sorry for completely dropping the ball on this one. Is this still good? Looks like you added tests a while back.

**Comment by alexjp** [comment]
@jeregrine Hi, 

Sorry, I will have to take another look and testing on this merge commit. Can I have some time to test it against latest elixir and phoenix ?

**Comment by jeregrine** [comment]
@alexjp take as much time as you'd like :-)

**Comment by jeregrine** [comment]
@alexjp FYI I've made you a collaborator on this repo. You've been very helpful <3

File: lib/mix/tasks/jsonapi.gen.resource.ex [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  def run(args) do
+    switches = [binary_id: :boolean, model: :boolean]
+
+    {opts, parsed, _} = OptionParser.parse(args, switches: switches)
+    [singular, plural | attrs] = validate_args!(parsed)
+
+    default_opts = Application.get_env(:phoenix, :generators, [])
+    opts = Keyword.merge(default_opts, opts)
+
+    attrs   = Mix.Phoenix.attrs(attrs)
```
Comment (doomspork) [inline lib/mix/tasks/jsonapi.gen.resource.ex]: Does this make Phoenix a requirement for `jsonapi` now?

**Pull request closed without merging**